src/lazy/expanded/compiler.rs

```diff
@@ -146,7 +174,7 @@ pub struct TemplateCompiler {}
 
 impl TemplateCompiler {
     /// Takes a TDL expression in the form:
-    /// ```ion_1_1
+    /// ```ion
```

Comment:
The `rustdoc` renderer doesn't differentiate between these two identifiers, but now that the template compiler is Ion-version-agnostic, I wanted this fence to be too.